### PR TITLE
Text utilities

### PR DESCRIPTION
**Pull request by Veekshith-Thoram**
image

### CODE REVIEW

File: text-utilities/text-utilities.css [hunk 1]
```diff
@@ -0,0 +1,125 @@
+@import url('https://fonts.googleapis.com/css2?family=Fredoka&display=swap');
+
+:root {
+    --primary-color: #34D399;
+
```
Comment (DevRaahul) [inline text-utilities/text-utilities.css]: remove this space

File: text-utilities/text-utilities.css [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+}
+
+
```
Comment (DevRaahul) [inline text-utilities/text-utilities.css]: try prettier extension it will format document on save

File: text-utilities/text-utilities.css [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+}
+
+#gray-text {
+    color: gray;
+}
+
+#center-text {
+    text-align: center;
+    font-weight: bold;
+}
```
Comment (DevRaahul) [inline text-utilities/text-utilities.css]: add empty line at end it will avoid such warning

File: text-utilities/text-utilities.css [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+.heading-text {
+    font-size: 1.3rem;
+    font-weight: bold;
+}
+
+.links {
+    text-decoration: none;
+    color: black;
```
Comment (DevRaahul) [inline text-utilities/text-utilities.css]: use constant for color here

File: text-utilities/text-utilities.css [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+}
+
+.main-container {
+    position: absolute;
+    left: 10rem;
+}
+
+.navigation {
+    background-color: var(--primary-color);
+    color: white;
```
Comment (DevRaahul) [inline text-utilities/text-utilities.css]: use color constant

File: text-utilities/text-utilities.css [hunk 6]
```diff
@@ -0,0 +1,125 @@
+@import url('https://fonts.googleapis.com/css2?family=Fredoka&display=swap');
+
+:root {
+    --primary-color: #34D399;
```
Comment (DevRaahul) [inline text-utilities/text-utilities.css]: use constant for color

**Review by DevRaahul — COMMENTED** [review]
Use prettier it will make your code more readable
Use color constant

**Review by NagaSatyaGeeth — APPROVED** [review]

**Pull request merged**